_targets.R

```diff
@@ -9,23 +9,27 @@ tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse b
 p1_targets_list <- list(
   tar_target(
     site_1_data_csv,
-    download_nwis_site_data(site_num = "01427207", fileout = "1_fetch/out/nwis_01427207_data.csv")
+    download_nwis_site_data(site_num = "01427207", fileout = "1_fetch/out/nwis_01427207_data.csv"),
+    format = "file"
```

Comment:
GREAT CATCH! 🎣 I completely missed that you didn't include `format = 'file'` in my initial review!